assets/src/hooks/use-copy-ticket.js

```diff
@@ -14,31 +14,34 @@ import useCreateRelationsForTicketToEventDates
 import useCreateRelationsForTicketToPrices
 	from './use-create-relations-for-ticket-to-prices';
 
+const PRICE = 'price';
+const PRICES = 'prices';
+const TICKET = 'ticket';
```

Comment:
@tn3rb let me know if I can pull these somewhere in a shared location.
Or maybe it's not so good idea.

Reply:
Yep, I totally agree with you, this change is out of the scope. I had reverted back those changes.
Thanks for the in-depth explanations, it clarified the situation a little bit for me :)